openid-connect-4-credential-issuance-1_0.md

```diff
@@ -469,13 +619,12 @@ A Client makes a Credential Request by presenting the following parameters:
 * `format`: OPTIONAL. Format of the credential to be issued. If not present, the issuer will determine the credential 
 format based on the clients format default. 
 * `sub_jwk`: CONDIIONAL. the key material the new credential shall be bound to.  
-* `did`: CONDIIONAL. the DID the credential shall be bound to. `sub_jwk` and `did` are mutual exclusive. 
+* `did`: CONDIIONAL. the DID the credential shall be bound to. `sub_jwk` and `did` are mutually exclusive. 
 * `proof` CONDITIONAL. JSON Object containing the proof of possession of the key material the issued credential shall be 
 bound to. The `proof` structure depends on the proof type. At the minimum, the following parameters MUST be included:
 
   * `type`: REQUIRED. JSON String denoting the proof type.
```

Comment:
Alen: what is type?
A: Inspired by vc-data-model proof section https://www.w3.org/TR/vc-data-model/#proofs-signatures

Reply:
what does jws signs? 
when JWS-VC, sign entire `proof` as a JWS.
Anoncreds have it pre-defined.